WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostPreviewFragment.java

```diff
@@ -32,7 +37,7 @@ public static EditPostPreviewFragment newInstance(SiteModel site, PostModel post
         EditPostPreviewFragment fragment = new EditPostPreviewFragment();
         Bundle bundle = new Bundle();
         bundle.putSerializable(WordPress.SITE, site);
-        bundle.putSerializable(EditPostActivity.EXTRA_POST, post);
+        bundle.putInt(EditPostActivity.EXTRA_POST_LOCAL_ID, post.getId());
```

Comment:
> It's no more confusing than having the remote and local id versions of getters in our stores.

In the stores, we don't really have the choice because remote ids can't be unique. So we make that distinction (remoteId vs localId) to avoid any ambiguity. 

But here in the app, we want to hide this and make it easier for developers. If we can avoid any id, that's even better. But in some cases (like here) we don't really have the choice. 

Also note: when FluxC is complete, we will never see any remote id in the app - we currently have some because we still have REST calls not going through FluxC.